.gitignore

```diff
@@ -7,3 +7,5 @@ doc
 pkg
 spec/reports
 tmp
+.idea
+.rakeTasks
```

Comment:
These seem like application-specific/custom settings? These should probably go in your local system’s gitignore instead of here.

Reply:
Alright. I'll fix up and push

Reply:
@christopherstyles PR Updated.